### PR TITLE
forbes press blog post

### PR DESCRIPTION
**Pull request by smombartz**
Forbes press blog post

screencapture-localhost-3002-blog-forbes-in-the-news-2020-01-03-14_26_40

### CODE REVIEW

**Comment by smombartz** [comment]
@akeem they are tweet embeds but i guess our styling is different...

**Comment by cnasc** [comment]
That does look like the issue:
> An embedded Tweet’s fallback markup will inherit the CSS rules already applied to `<blockquote>`, `<p>`, and `<a>` HTML elements on your webpage.

from https://developer.twitter.com/en/docs/twitter-for-websites/embedded-tweets/guides/css-for-embedded-tweets

**Comment by julien51** [comment]
Just pushed an update with styling fixed ;)

image

File: unlock-protocol.com/src/components/content/BlogPost.js [hunk 1]
```diff
@@ -9,19 +9,37 @@ export class BlogPost extends React.Component {
...
+    const { scripts } = this.props
+    this.scripts = scripts.map(url => {
+      const element = document.createElement('script')
+      element.async = true
+      element.src = url
+      element.charset = 'utf-8'
+      document.body.appendChild(element)
+      return element
+    })
+    window.julien = this.scripts
```
Comment (cnasc) [inline unlock-protocol.com/src/components/content/BlogPost.js]: I think you left a debug line in here

Comment (julien51) [reply unlock-protocol.com/src/components/content/BlogPost.js]: Oupsie :/

**Pull request merged**